meetings/2025-01-29.md

```diff
@@ -0,0 +1,43 @@
+# Node.js  TypeScript team Meeting 2025-01-29
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=9Wa0FBTxFN8>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/26>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Jake Bailey: @jakebailey
+* Lucas Santos: @khaosdoctor
+* Augustin Mauroy @AugustinMauroy
+
+## Agenda
+
+## Announcements
+
+Jake: Typescript 5.8 –erasableSyntaxOnly out, Node.js needs to do some changes in the swc implementation to match the ts behavior on namespaces, test cases can be copied from <https://github.com/bloomberg/ts-blank-space/pull/32>
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/typescript
+
+* Roadmap to stable strip-types [#24](https://github.com/nodejs/typescript/issues/24)
+Marco:
+* working improving errors, currently blocked by swc.
+* need to create a tsconfig for users
+Jake: probably better to add it to <https://github.com/tsconfig/bases> instead of creating a CLI tool -> Lucas volunteered (wait until 5.8 is out on beta and state that it will only work for the beta in the description)
+Lucas: maybe ask the maintainers if anyone in the Node.js TS team can join the org to maintain the Node.js specific files
+Marco: plan to unflag on v22 after a couple more releases v23 releases.
+Augustin: should we suggest in which cases users should use strip-types
+Marco: we should wait for stabilization before recommending something
+Marco: things going smoothly so far, no major blockers
```

Comment:
These render weird; probably need to prefix with `*` to make it a list rather than one paragraph.